script/Ciel/Day 10/10_01_CIEL10_9.txt

```diff
@@ -831,7 +831,7 @@ Even now, she clings to that oath.
 [sha:70a5e78b16c2c7703188f514790225443c66336a]{
 -- Page 650, Offset 28927.
 -- 吸血鬼への復讐こそが彼女の生存理由だったからだ。
-After all, taking revenge on vampires had been her raison d'être.
+After all, taking revenge on vampires had been her %{i}raison d'être%{/i}.
```

Comment:
Similarly to force majeure, this is an ordinary loan phrase, not sure it deserves italics.